docs/libtorch.rst

```diff
@@ -34,3 +34,18 @@ To produce libtorch.a rather than libtorch.so, set the environment variable `BUI
 To use ninja rather than make, set `CMAKE_GENERATOR="-GNinja" CMAKE_INSTALL="ninja install"`.
 
 Note that we are working on eliminating tools/build_pytorch_libs.sh in favor of a unified cmake build.
+
+Building libtorch without using Python
+--------------------------------------
+
+You can build C++ libtorch.so without relying on Python.  For example to build a v1.6.0 Release version that uses (system-installed) CUDA and CUDDN, and install it in the directory specified by CMAKE_INSTALL_PREFIX below, you can use
+::
+   git clone https://github.com/pytorch/pytorch.git
+   cd pytorch
+   git checkout v1.6.0
+   git clean -fdx
+   git submodule update --init --recursive
```

Comment:
Why one need to call `clean -fdx` on a freshly checked out repo?
Also, in order to avoid changing documentation with every release, it probably better to document how to build a latest master
```suggestion
   git clone --recurse-submodules https://github.com/pytorch/pytorch.git
   cd pytorch
```

Reply:
The initial `clone` as written will bring in the `master` branch.  The checkout of an earlier tag will make changed files revert to their version at the time of the tag, but won't always delete files and directories that were not present for that tag.  The `git clean -fdx` covers us for that case, so that the subsequent build won't try to use files or directories that shouldn't be there.

There is value in showing how to do this with `master` but many users won't want the bleeding edge implementation, rather a stable tagged version.  We could show both.  I'll put in a commit with that.

Reply:
Please see the most recent commit.

Reply:
`git clone -b` can be used to checkout specific tag/branch.
My concerns with referencing specific release tag is that it will require updating the README with every release.
Besides there are already binary package for release tags  available at https://www.pytorch.org, so there isn't much added value in rebuilding them locally, is there?